include/irr/builtin/glsl/sampling/cos_weighted.glsl

```diff
@@ -40,4 +45,9 @@ float irr_glsl_projected_sphere_remainder_and_pdf(out float pdf, in vec3 L)
     return irr_glsl_projected_sphere_remainder_and_pdf(pdf,L.z);
 }
 
+float irr_glsl_projected_sphere_pdf(in float L_z)
```

Comment:
same thing as https://github.com/buildaworldnet/IrrlichtBAW/pull/554/files#r518968449

Reply:
duplicate code

Reply:
still duplicate code... I only want one place where the following code exists `L_z * irr_glsl_RECIPROCAL_PI`